core/src/main/java/org/apache/carbondata/core/scan/filter/executer/FilterExecuter.java

```diff
@@ -46,7 +46,7 @@ boolean applyFilter(RowIntf value, int dimOrdinalMax)
    * @param blockMinValue
    * @return BitSet
    */
-  BitSet isScanRequired(byte[][] blockMaxValue, byte[][] blockMinValue);
+  BitSet isScanRequired(byte[][] blockMaxValue, byte[][] blockMinValue, byte[] blockNullValue);
```

Comment:
Better wrap these parameters to new BlockletStats object

Reply:
Done